product_move_analytical_account/models/stock_move_line.py

```diff
@@ -0,0 +1,18 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import api, fields, models
+
+class AccountAnalyticAccount(models.Model):
+    _inherit = "stock.move.line"
+
+    account_analytic_id = fields.Many2one('account.analytic.account',
+        string="Account Analytic",
+        compute="_compute_account_analytic",
+        store=True)
```

Comment:
Better practice:
```suggestion
    po_account_analytic_id = fields.Many2one('account.analytic.account',
        string="Analytic Account",
        compute="_compute_po_account_analytic_id",
        store=True)
```
* Prefix field name (a stock move line could belong to a sale order line, and sale order lines could apply their own analytic accounts in the future. Anyway, we can easily identify the field origin with this trick).
* Better `string` for the field in english.
* compute function should be named `_compute_<field_name>` if possible.

Reply:
Fixes done, and squashed commits.